src/styles/pages/main/main.scss

```diff
@@ -0,0 +1,377 @@
+.header {
+
+    &__logo {
+        padding: 9px 0;
+    }
+
+    .logo {
+        font-size: 35px;
+        color: white;
+        font-family: "Dancing Script", cursive;
+        border-bottom: 1px solid #fff;
+    } 
+
+    .container {
```

Comment:
move the container separately from the header

Reply:
you should move the header in general stules, becose you'll add header to another pages.
and you should create general styles and add some common styles there(for example: container, buttons and so on)